index.js

```diff
@@ -0,0 +1,60 @@
+const Storage = {
+  fetch() {
+    return JSON.parse(localStorage.getItem('todoLists')) || []
+  },
+  save(todoLists) {
+    localStorage.setItem('todoLists', JSON.stringify(todoLists))
+  }
+}
+
+const app = new Vue({
+  el: '#app',
+  data: {
+    todoLists: Storage.fetch(),
+    text: '',
+    currentIndex: ''
+  },
+  methods: {
+    addText: function() {
```

Comment:
`Storage` でやっているのと同じように、省略記法で書いてもよいかもですね！ (他のメソッドも同様)

```suggestion
    addText() {
```